### PR TITLE
autoscaler: validate git history

### PR DESCRIPTION
**Pull request by paulfantom**
Same as #3116.

/assign @frobware

### CODE REVIEW

**Comment by paulfantom** [comment]
`ci/rehearse/openshift/kubernetes-autoscaler/master/git-history` rehersal job should fail as last commit is not prefixed.

File: ci-operator/config/openshift/kubernetes-autoscaler/openshift-kubernetes-autoscaler-master.yaml [hunk 1]
```diff
@@ -65,3 +65,18 @@ tests:
     make test-e2e
   openshift_installer_src:
     cluster_profile: aws
+- as: git-history
+  commands: |
+    PREFIX="UPSTREAM: <carry>: openshift:"
```
Comment (ingvagabund) [inline ci-operator/config/openshift/kubernetes-autoscaler/openshift-kubernetes-autoscaler-master.yaml]: Can you do the same (i.e. `PREFIX="UPSTREAM: <[[:alnum:]]+>: openshift:"`) as in https://github.com/openshift/release/pull/3116/files#diff-f0e4e08ecb398ef3dfe3a832e3714cd7R59 ?

Comment (paulfantom) [reply ci-operator/config/openshift/kubernetes-autoscaler/openshift-kubernetes-autoscaler-master.yaml]: Did it locally and forgot to push changes :man_facepalming:

**Comment by ingvagabund** [comment]
Job is optinal, thus non-blocking

**Comment by paulfantom** [comment]
I wonder if this will work
/approve

**Comment by paulfantom** [comment]
@droslean could you approve this? We already changed OWNERS file, but this PR was created before changing OWNERS and it seems that tide still respects old settings.

**Pull request merged**